src/models/components.d.ts

```diff
@@ -0,0 +1,64 @@
+import { SvgIconComponent } from "@mui/icons-material";
+
+/*=============================================
+=            // 1,  factSearchLists            =
+=============================================*/
+
+export interface listItemTypes {
```

Comment:
Your types should be named according to updated component names.

Reply:
done

Reply:
@Ayomikuolatunji Again spelling mistake in this file.

```
export interface catergoryItemType {
  label: string;
  Icon: SvgIconComponent;
}
```